lib/qa/authorities/linked_data/rdf_helper.rb

```diff
@@ -0,0 +1,82 @@
+require 'rdf'
+
+# Encapsulates processing of RDF results returned by the linked data authority.  This is used exclussively by Qa::Authorities::LinkedData::GenericAuthority
+# @see Qa::Authorities::LinkedData::GenericAuthority
+module Qa::Authorities
+  module LinkedData
+    module RdfHelper
```

Comment:
this seems like a good suggestion but I know @elrayle is ready to get this merged. Maybe an issue targeting this as a refactor task?